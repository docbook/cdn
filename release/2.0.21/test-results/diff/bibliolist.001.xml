<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: bibliolist.001</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: bibliolist.001</h2>
         </header>
         <p deltaxml:deltaV2="A">This is a article.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">bibliolist</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class><dxa:id deltaxml:deltaV2="A"><deltaxml:attributeValue deltaxml:deltaV2="A">R.1.3</deltaxml:attributeValue></dxa:id></deltaxml:attributes>
            <h3 deltaxml:deltaV2="A">Bibliography List</h3><p deltaxml:deltaV2="B">This is a article.</p>
            <div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">biblioentry</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">bibliolist</deltaxml:attributeValue></dxa:class><dxa:id deltaxml:deltaV2="B"><deltaxml:attributeValue deltaxml:deltaV2="B">R.1.3</deltaxml:attributeValue></dxa:id></deltaxml:attributes>
               <p deltaxml:deltaV2="A" class="biblioentry">
                  <span class="label">[<span class="abbrev">AhoSethiUllman96</span>] </span>
                  <span class="entry">
                     <span class="authorgroup">
                        <span class="firstname">Alfred V.</span>
                        <span class="surname">Aho</span>, <span class="firstname">Ravi</span>
                        <span class="surname">Sethi</span>, and <span class="firstname">Jeffrey D.</span>
                        <span class="surname">Ullman</span>
                     </span>. <span class="title">
                        <cite>Compilers, Principles, Techniques, and Tools</cite>
                     </span>. <span class="publishername">Addison-Wesley Publishing Company</span>. <span class="copyright">Copyright © <span class="years">1996</span> <span class="holders">
                           <span class="holder">Bell Telephone Laboratories, Inc.</span>
                        </span>
                     </span>
                     <span class="biblioid">0-201-10088-6</span>. <span class="editor">
                        <span class="firstname">James T.</span>
                        <span class="surname">DeWolf</span>
                     </span>. </span>
               </p><h3 deltaxml:deltaV2="B">Bibliography List</h3><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="biblioentry">
                  <p class="biblioentry">
                     <span class="label">[<span class="abbrev">AhoSethiUllman96</span>] </span>
                     <span class="entry">
                        <span class="authorgroup">
                           <span class="firstname">Alfred V.</span>
                           <span class="surname">Aho</span>, <span class="firstname">Ravi</span>
                           <span class="surname">Sethi</span>, and <span class="firstname">Jeffrey D.</span>
                           <span class="surname">Ullman</span>
                        </span>. <span class="title">
                           <cite>Compilers, Principles, Techniques, and Tools</cite>
                        </span>. <span class="publishername">Addison-Wesley Publishing Company</span>. <span class="copyright">Copyright © <span class="years">1996</span> <span class="holders">
                              <span class="holder">Bell Telephone Laboratories, Inc.</span>
                           </span>
                        </span>
                        <span class="biblioid">0-201-10088-6</span>. <span class="editor">
                           <span class="firstname">James T.</span>
                           <span class="surname">DeWolf</span>
                        </span>. </span>
                  </p>
               </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
               </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" id="Walsh97" class="biblioentry">
                  <p class="biblioentry">
                     <span class="label">[<span class="abbrev">Walsh97</span>] </span>
                     <span class="entry">
                        <span class="biblioset">“A Guide to XML”. <span class="author">
                              <span class="firstname">Norman</span>
                              <span class="surname">Walsh</span>
                           </span>. <span class="pubdate">1997</span>. <span class="copyright">Copyright © <span class="years">1997</span> <span class="holders">
                                 <span class="holder">ArborText, Inc.</span>
                              </span>
                           </span>
                           <span class="pagenums">97-108</span>. </span>
                        <span class="biblioset">
                           <cite>XML: Principles, Tools, and Techniques</cite>. <span class="publishername">O'Reilly &amp; Associates, Inc.</span>
                           <span class="biblioid">1085-2301</span>. <span class="editor">
                              <span class="firstname">Dan</span>
                              <span class="surname">Connolly</span>
                           </span>. </span>
                     </span>
                  </p>
               </div>
            </div>
            <div deltaxml:deltaV2="A" id="Walsh97" class="biblioentry">
               <p class="biblioentry">
                  <span class="label">[<span class="abbrev">Walsh97</span>] </span>
                  <span class="entry">
                     <span class="biblioset">“A Guide to XML”. <span class="author">
                           <span class="firstname">Norman</span>
                           <span class="surname">Walsh</span>
                        </span>. <span class="pubdate">1997</span>. <span class="copyright">Copyright © <span class="years">1997</span> <span class="holders">
                              <span class="holder">ArborText, Inc.</span>
                           </span>
                        </span>
                        <span class="pagenums">97-108</span>. </span>
                     <span class="biblioset">
                        <cite>XML: Principles, Tools, and Techniques</cite>. <span class="publishername">O'Reilly &amp; Associates, Inc.</span>
                        <span class="biblioid">1085-2301</span>. <span class="editor">
                           <span class="firstname">Dan</span>
                           <span class="surname">Connolly</span>
                        </span>. </span>
                  </span>
               </p>
            </div><p deltaxml:deltaV2="B">This is another paragraph.</p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">This is another paragraph.</p>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>