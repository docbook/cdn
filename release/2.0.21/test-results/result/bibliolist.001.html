<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for bibliolist.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: bibliolist.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">10</span> differences)</th><th>Actual (<span class="adiff">10</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: bibliolist.001&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: bibliolist.001&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;This is a article.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">bibliolist</span>"</span> <span class="att add">id="<span class="att">R.1.3</span>"</span>&gt;
            <span class="tag diff del">&lt;h3&gt;Bibliography List&lt;/h3&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">biblioentry</span>"</span> <span class="att del">id="<span class="att"></span>"</span>&gt;
               <span class="tag diff del">&lt;p <span class="att">class="biblioentry"</span>&gt;
                  <span class="tag">&lt;span <span class="att">class="label"</span>&gt;[<span class="tag">&lt;span <span class="att">class="abbrev"</span>&gt;AhoSethiUllman96&lt;/span&gt;</span>] &lt;/span&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="entry"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="authorgroup"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Alfred V.&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Aho&lt;/span&gt;</span>, <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Ravi&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Sethi&lt;/span&gt;</span>, and <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Jeffrey D.&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Ullman&lt;/span&gt;</span>
                     &lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="title"</span>&gt;
                        <span class="tag">&lt;cite&gt;Compilers, Principles, Techniques, and Tools&lt;/cite&gt;</span>
                     &lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="publishername"</span>&gt;Addison-Wesley Publishing Company&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="copyright"</span>&gt;Copyright © <span class="tag">&lt;span <span class="att">class="years"</span>&gt;1996&lt;/span&gt;</span> <span class="tag">&lt;span <span class="att">class="holders"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="holder"</span>&gt;Bell Telephone Laboratories, Inc.&lt;/span&gt;</span>
                        &lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="biblioid"</span>&gt;0-201-10088-6&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="editor"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;James T.&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;DeWolf&lt;/span&gt;</span>
                     &lt;/span&gt;</span>. &lt;/span&gt;</span>
               &lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag diff del">&lt;div <span class="att">class="biblioentry"</span> <span class="att">id="Walsh97"</span>&gt;
               <span class="tag">&lt;p <span class="att">class="biblioentry"</span>&gt;
                  <span class="tag">&lt;span <span class="att">class="label"</span>&gt;[<span class="tag">&lt;span <span class="att">class="abbrev"</span>&gt;Walsh97&lt;/span&gt;</span>] &lt;/span&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="entry"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="biblioset"</span>&gt;“A Guide to XML”. <span class="tag">&lt;span <span class="att">class="author"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Norman&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Walsh&lt;/span&gt;</span>
                        &lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="pubdate"</span>&gt;1997&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="copyright"</span>&gt;Copyright © <span class="tag">&lt;span <span class="att">class="years"</span>&gt;1997&lt;/span&gt;</span> <span class="tag">&lt;span <span class="att">class="holders"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="holder"</span>&gt;ArborText, Inc.&lt;/span&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="pagenums"</span>&gt;97-108&lt;/span&gt;</span>. &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="biblioset"</span>&gt;
                        <span class="tag">&lt;cite&gt;XML: Principles, Tools, and Techniques&lt;/cite&gt;</span>. <span class="tag">&lt;span <span class="att">class="publishername"</span>&gt;O'Reilly &amp; Associates, Inc.&lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="biblioid"</span>&gt;1085-2301&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="editor"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Dan&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Connolly&lt;/span&gt;</span>
                        &lt;/span&gt;</span>. &lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;This is another paragraph.&lt;/p&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: bibliolist.001&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: bibliolist.001&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span> <span class="att del">id="<span class="att"></span>"</span>&gt;
            <span class="tag diff add">&lt;p&gt;This is a article.&lt;/p&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">bibliolist</span>"</span> <span class="att add">id="<span class="att">R.1.3</span>"</span>&gt;
               <span class="tag diff add">&lt;h3&gt;Bibliography List&lt;/h3&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="biblioentry"</span>&gt;
                  <span class="tag">&lt;p <span class="att">class="biblioentry"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="label"</span>&gt;[<span class="tag">&lt;span <span class="att">class="abbrev"</span>&gt;AhoSethiUllman96&lt;/span&gt;</span>] &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="entry"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="authorgroup"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Alfred V.&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Aho&lt;/span&gt;</span>, <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Ravi&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Sethi&lt;/span&gt;</span>, and <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Jeffrey D.&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Ullman&lt;/span&gt;</span>
                        &lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="title"</span>&gt;
                           <span class="tag">&lt;cite&gt;Compilers, Principles, Techniques, and Tools&lt;/cite&gt;</span>
                        &lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="publishername"</span>&gt;Addison-Wesley Publishing Company&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="copyright"</span>&gt;Copyright © <span class="tag">&lt;span <span class="att">class="years"</span>&gt;1996&lt;/span&gt;</span> <span class="tag">&lt;span <span class="att">class="holders"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="holder"</span>&gt;Bell Telephone Laboratories, Inc.&lt;/span&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="biblioid"</span>&gt;0-201-10088-6&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="editor"</span>&gt;
                           <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;James T.&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;DeWolf&lt;/span&gt;</span>
                        &lt;/span&gt;</span>. &lt;/span&gt;</span>
                  &lt;/p&gt;</span>
               &lt;/div&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="biblioentry"</span> <span class="att">id="Walsh97"</span>&gt;
                  <span class="tag">&lt;p <span class="att">class="biblioentry"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="label"</span>&gt;[<span class="tag">&lt;span <span class="att">class="abbrev"</span>&gt;Walsh97&lt;/span&gt;</span>] &lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="entry"</span>&gt;
                        <span class="tag">&lt;span <span class="att">class="biblioset"</span>&gt;“A Guide to XML”. <span class="tag">&lt;span <span class="att">class="author"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Norman&lt;/span&gt;</span>
                              <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Walsh&lt;/span&gt;</span>
                           &lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="pubdate"</span>&gt;1997&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="copyright"</span>&gt;Copyright © <span class="tag">&lt;span <span class="att">class="years"</span>&gt;1997&lt;/span&gt;</span> <span class="tag">&lt;span <span class="att">class="holders"</span>&gt;
                                 <span class="tag">&lt;span <span class="att">class="holder"</span>&gt;ArborText, Inc.&lt;/span&gt;</span>
                              &lt;/span&gt;</span>
                           &lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="pagenums"</span>&gt;97-108&lt;/span&gt;</span>. &lt;/span&gt;</span>
                        <span class="tag">&lt;span <span class="att">class="biblioset"</span>&gt;
                           <span class="tag">&lt;cite&gt;XML: Principles, Tools, and Techniques&lt;/cite&gt;</span>. <span class="tag">&lt;span <span class="att">class="publishername"</span>&gt;O'Reilly &amp; Associates, Inc.&lt;/span&gt;</span>
                           <span class="tag">&lt;span <span class="att">class="biblioid"</span>&gt;1085-2301&lt;/span&gt;</span>. <span class="tag">&lt;span <span class="att">class="editor"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Dan&lt;/span&gt;</span>
                              <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Connolly&lt;/span&gt;</span>
                           &lt;/span&gt;</span>. &lt;/span&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag diff add">&lt;p&gt;This is another paragraph.&lt;/p&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: bibliolist.001</h2></header><p>This is a article.</p><div id="R.1.3" class="bibliolist"><h3>Bibliography List</h3>


<div class="biblioentry"><p class="biblioentry"><span class="label">[<span class="abbrev">AhoSethiUllman96</span>] </span><span class="entry"><span class="authorgroup"><span class="firstname">Alfred V.</span> <span class="surname">Aho</span>, <span class="firstname">Ravi</span> <span class="surname">Sethi</span>, and <span class="firstname">Jeffrey D.</span> <span class="surname">Ullman</span></span>. <span class="title"><cite>Compilers, Principles, Techniques, and Tools</cite></span>. <span class="publishername">Addison-Wesley Publishing Company</span>. <span class="copyright">Copyright © <span class="years">1996</span> <span class="holders"><span class="holder">Bell Telephone Laboratories, Inc.</span></span></span> <span class="biblioid">0-201-10088-6</span>. <span class="editor"><span class="firstname">James T.</span> <span class="surname">DeWolf</span></span>. </span></p></div>

<div id="Walsh97" class="biblioentry"><p class="biblioentry"><span class="label">[<span class="abbrev">Walsh97</span>] </span><span class="entry"><span class="biblioset">“A Guide to XML”. <span class="author"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span>. <span class="pubdate">1997</span>. <span class="copyright">Copyright © <span class="years">1997</span> <span class="holders"><span class="holder">ArborText, Inc.</span></span></span> <span class="pagenums">97-108</span>. </span><span class="biblioset"><cite>XML: Principles, Tools, and Techniques</cite>. <span class="publishername">O'Reilly &amp; Associates, Inc.</span> <span class="biblioid">1085-2301</span>. <span class="editor"><span class="firstname">Dan</span> <span class="surname">Connolly</span></span>. </span></span></p></div>
</div><p>This is another paragraph.</p></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: bibliolist.001</h2></header><div class="content"><p>This is a article.</p><div id="R.1.3" class="bibliolist"><h3>Bibliography List</h3>


<div class="biblioentry"><p class="biblioentry"><span class="label">[<span class="abbrev">AhoSethiUllman96</span>] </span><span class="entry"><span class="authorgroup"><span class="firstname">Alfred V.</span> <span class="surname">Aho</span>, <span class="firstname">Ravi</span> <span class="surname">Sethi</span>, and <span class="firstname">Jeffrey D.</span> <span class="surname">Ullman</span></span>. <span class="title"><cite>Compilers, Principles, Techniques, and Tools</cite></span>. <span class="publishername">Addison-Wesley Publishing Company</span>. <span class="copyright">Copyright © <span class="years">1996</span> <span class="holders"><span class="holder">Bell Telephone Laboratories, Inc.</span></span></span> <span class="biblioid">0-201-10088-6</span>. <span class="editor"><span class="firstname">James T.</span> <span class="surname">DeWolf</span></span>. </span></p></div>

<div id="Walsh97" class="biblioentry"><p class="biblioentry"><span class="label">[<span class="abbrev">Walsh97</span>] </span><span class="entry"><span class="biblioset">“A Guide to XML”. <span class="author"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span>. <span class="pubdate">1997</span>. <span class="copyright">Copyright © <span class="years">1997</span> <span class="holders"><span class="holder">ArborText, Inc.</span></span></span> <span class="pagenums">97-108</span>. </span><span class="biblioset"><cite>XML: Principles, Tools, and Techniques</cite>. <span class="publishername">O'Reilly &amp; Associates, Inc.</span> <span class="biblioid">1085-2301</span>. <span class="editor"><span class="firstname">Dan</span> <span class="surname">Connolly</span></span>. </span></span></p></div>
</div><p>This is another paragraph.</p></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/bibliolist.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>