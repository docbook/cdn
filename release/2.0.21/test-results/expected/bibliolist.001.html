<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: bibliolist.001</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Unit Test: bibliolist.001</h2></header><p>This is a article.</p><div id="R.1.3" class="bibliolist"><h3>Bibliography List</h3>


<div class="biblioentry"><p class="biblioentry"><span class="label">[<span class="abbrev">AhoSethiUllman96</span>] </span><span class="entry"><span class="authorgroup"><span class="firstname">Alfred V.</span> <span class="surname">Aho</span>, <span class="firstname">Ravi</span> <span class="surname">Sethi</span>, and <span class="firstname">Jeffrey D.</span> <span class="surname">Ullman</span></span>. <span class="title"><cite>Compilers, Principles, Techniques, and Tools</cite></span>. <span class="publishername">Addison-Wesley Publishing Company</span>. <span class="copyright">Copyright © <span class="years">1996</span> <span class="holders"><span class="holder">Bell Telephone Laboratories, Inc.</span></span></span> <span class="biblioid">0-201-10088-6</span>. <span class="editor"><span class="firstname">James T.</span> <span class="surname">DeWolf</span></span>. </span></p></div>

<div id="Walsh97" class="biblioentry"><p class="biblioentry"><span class="label">[<span class="abbrev">Walsh97</span>] </span><span class="entry"><span class="biblioset">“A Guide to XML”. <span class="author"><span class="firstname">Norman</span> <span class="surname">Walsh</span></span>. <span class="pubdate">1997</span>. <span class="copyright">Copyright © <span class="years">1997</span> <span class="holders"><span class="holder">ArborText, Inc.</span></span></span> <span class="pagenums">97-108</span>. </span><span class="biblioset"><cite>XML: Principles, Tools, and Techniques</cite>. <span class="publishername">O'Reilly &amp; Associates, Inc.</span> <span class="biblioid">1085-2301</span>. <span class="editor"><span class="firstname">Dan</span> <span class="surname">Connolly</span></span>. </span></span></p></div>
</div><p>This is another paragraph.</p></article><script src="../../resources/js/prism.js"></script></body></html>